TransferMethod/Sources/SelectTransferMethodTypePresenter.swift

```diff
@@ -141,6 +143,8 @@ final class SelectTransferMethodTypePresenter {
             if case let .success(user) = getUserResult,
                 let profileType = user?.profileType?.rawValue {
                 let transferMethodTypeCode = strongSelf.sectionData[index].code!
+                self?.selectedTransferMethodType = transferMethodTypeCode
```

Comment:
I know ``` let transferMethodTypeCode = strongSelf.sectionData[index].code!```  is existing code but refactor it use if let instead of force unwrap (!)

Reply:
If let added